wordpress-mobile/WordPress-iOS
# Add Manage Screen to the new Reader tab

Adds Manage Screen (#13582   )

## Code Changes

- Adds `TabbedViewController`, which is a generic version of a container view with tabs to navigate between each view controller. This View Controller doesn't currently have any fancy lazy loading.
- Adds a new `ReaderTagsTableViewController` which uses `WPTableViewHandler` instead of reusing `ReaderMenuViewController`.
- Reuses `ReaderFollowedSitesViewController` for the followed sites, with some tweaks for styling.

## To Test
- Open Reader Tab
- Tap the "Cog" icon to open the "Manage" screen
- Check that Tags are followed / unfollowed as expected
- Check that Sites are followed / unfollowed as expected

## Screenshots

<img width="300" alt="Screen Shot 2020-04-19 at 5 01 51 PM" src="https://user-images.githubusercontent.com/3250/79768998-cce7fa80-82e8-11ea-9604-d78026a312a9.png"><img width="300" alt="Screen Shot 2020-04-19 at 5 01 56 PM" src="https://user-images.githubusercontent.com/3250/79769010-d1acae80-82e8-11ea-9fe4-e736abd61f64.png">

<details>
  <summary><strong>iPad Screenshots</strong></summary>

<img width="873" alt="Screen Shot 2020-04-19 at 5 07 17 PM" src="https://user-images.githubusercontent.com/3250/79768971-c0fc3880-82e8-11ea-9d62-fe8ceb3d4036.png">
<img width="873" alt="Screen Shot 2020-04-19 at 5 07 08 PM" src="https://user-images.githubusercontent.com/3250/79769256-19cbd100-82e9-11ea-9243-e7ab138a09dd.png">

<img width="1151" alt="Screen Shot 2020-04-19 at 5 07 02 PM" src="https://user-images.githubusercontent.com/3250/79769089-ec7f2300-82e8-11ea-9358-6fe145d7cf6b.png">
<img width="1151" alt="Screen Shot 2020-04-19 at 5 07 05 PM" src="https://user-images.githubusercontent.com/3250/79769090-edb05000-82e8-11ea-90ee-663bc30c3cab.png">
</details>

PR submission checklist:

- [x] ~I have considered adding unit tests where possible.~ I have a set of these created but want to submit in a follow up PR.
